mayastor/src/rebuild.rs

```diff
@@ -168,14 +187,28 @@ impl RebuildJob {
             return Err(RebuildError::InvalidParameters {});
         };
 
-        let segment_size = 10 * 1024;
+        let segment_size = SEGMENT_SIZE;
```

Comment:
We could just use the constant directly.

Reply:
Done